cfgov/jinja2/v1/_includes/organisms/half-width-link-blob-group.html

```diff
@@ -0,0 +1,5 @@
+<div class="o-link-blob-group">
+    {% for block in value.link_blobs %}
```

Comment:
@KimberlyMunoz if they are always groups will they also have an optional group header like in the landing page?

```
{% if page.image_text_50_50_content %}
        <div class="o-image-text-50-50-group">
            <h2 class="o-image-text-50-50-group__header">
                {{ page.image_text_50_50_group_header }}
            </h2>

            {% for block in page.image_text_50_50_content %}
                {{ render_stream_child(block) }}
            {% endfor %}
        </div>
    {% endif %}
```